src/components/styled-password-input/styled-password-input.tsx

```diff
@@ -10,17 +11,28 @@ import { useStyledPasswordInputStyles } from './styled-password-input.styles';
 
 export const StyledPasswordInput: FC<StyledPasswordInputProps> = ({ value, ...props }) => {
   const [isSecureTextEntry, setIsSecureTextEntry] = useState(true);
+  const { activeTimer, clearActiveTimer } = useActiveTimer();
 
   const styles = useStyledPasswordInputStyles();
 
+  const handleSecureEntryPress = () => {
+    if (isSecureTextEntry) {
+      clearActiveTimer();
+      activeTimer.current = setTimeout(() => {
+        setIsSecureTextEntry(true);
+      }, 10_000);
```

Comment:
reuse `OVERLAY_SHOW_TIMEOUT` variable and make short return inside `setTimeout` callback please